extensions-core/druid-basic-security/src/main/java/org/apache/druid/security/basic/authentication/db/cache/CoordinatorPollingBasicAuthenticatorCacheManager.java

```diff
@@ -69,6 +70,7 @@
   private static final EmittingLogger LOG = new EmittingLogger(CoordinatorPollingBasicAuthenticatorCacheManager.class);
 
   private final ConcurrentHashMap<String, Map<String, BasicAuthenticatorUser>> cachedUserMaps;
+  private final ConcurrentHashMap<String, BasicAuthConfig> cachedConfigs;
```

Comment:
This class has a logic to write the userMaps to disk and recover from the disk. 
Looks like, we need same logic for cachedConfigs too.

Reply:
Here I was thinking that when the service gets started or restarted then we want the reading/loading of configs from the runtime properties file to always take precedence over configs from cachedConfigs.

Reply:
This is now irrelevant since we aren't polling and caching configs any longer, functionality removed